pymeasure/instruments/keithley/keithley2602B.py

```diff
@@ -22,6 +22,8 @@
 #
 
 import logging
+from socket import error as socket_error
```

Comment:
Could alternately `import socket` and write `socket.socket_error` below to be more explicit.